modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -34,7 +34,8 @@
 	--n-tabs-title-background-color: #{$editor-background-light};
 	--n-tabs-title-color-active: #{$white};
 	--n-tabs-title-background-color-active: var( --e-global-color-accent, var( --n-tabs-color-accent-fallback ) );
-	--n-tabs-title-width: initial;
+	--n-tabs-title-background-color-hover: var( --e-global-color-accent, var( --n-tabs-color-accent-fallback ) );
```

Comment:
Safe to delete?

Reply:
I will delete them once I'll see that all the backgrounds work well.
Good comment :)

Reply:
We have decided to leave it because we are using it to fill the lack of PHP global colors.